lib/astutils.cpp

```diff
@@ -853,6 +853,11 @@ static bool isScopeBracket(const Token *tok)
 }
 
 bool isVariableChangedByFunctionCall(const Token *tok, const Settings *settings, bool *inconclusive)
+{
```

Comment:
I think it would be better to have just 1 function... I assume you do it this way to avoid rewriting lots of code? Then we should rewrite the old function calls later in a separate commit.

Reply:
I believe it was mostly laziness so that I could avoid changing the existing calls :o. But I can fix that.

Reply:
For information I am not against doing this in 2 steps. But if you change all old calls now that is fine also.

Reply:
Ok, looks like this has been fixed by #2123. Thanks @pfultz2 !